<?xml version="1.0" ?>
<launch> 
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
    <rosparam command="load" file="$(find flaw_detection)/config/param.yaml" ns="accupick3d/flaw_detection/"/>
    
    <node name="arm_control" pkg="arm_control" type="control.py"/>
    
    <!--<node name="gomoku_strategy" pkg="gomoku" type="strategy"/>-->

    <!--<node name="serial_node" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM0"/>-->
</launch>
